### ISSUE TITLE
We need a way to obtain lead

### ISSUE THREAD

**Issue by UbaserB** [space-wizards/space-station-14#23728]
## Description

Something that is more difficult than steel probably, and it needs to make sense too. Otherwise, licoxide/tazinide is currently unobtainable. This issue has been floating around for ages but i decided to make a formal issue on github for it.

**Comment by deltanedas** [comment]
tazinide makes no sense:
- its traitor exclusive
- yet its a better validhunting reagent
- you could make nocturine or lexorin instead which is way better for actually doing objectives

**Comment by UbaserB** [comment]
> tazinide makes no sense:
> 
> * its traitor exclusive
> * yet its a better validhunting reagent
> * you could make nocturine or lexorin instead which is way better for actually doing objectives

I dont believe this is related to the issue i have posted. I am talking about lead being unobtainable.

**Comment by deltanedas** [comment]
still should be considered for leads availability

**Comment by Holinka4ever** [comment]
Struggling though google, i made some conclusion from where we should get lead:

1) grinding used bullet cassing for a mere amount, but it can still work out
2) batteries uhu

In first way it understandable, just shoot from a weapon and get lead,  due of curret state,its not difficut to get used bullet cassings

At second thing, its a bit of... complex
Tl;dr story.
First of all licoxide were a stun posion that you could make for free by grinding a battery. Dunno why it got into first place. Someone to order to balance it, turn that batteries gives zinc instead of licoxide. And lead is getting from metal sheet, that gave 0.5 unit, huh. 
But gladly we have knowledge, how it can turn the tides. Grinding condesators into lead because its some how but accumulator. People whould ask why, and i will answer that you can get copper from metal wires.

**Comment by GreyMario** [comment]
Licoxide is a Nukies chemical now. You obtain lead by emagging a Syndicate chemvend.
